cypress.json

```diff
@@ -1,6 +1,7 @@
 {
-  "screenshots": {
-    "screenshotOnRunFailure": false
-  },
-  "projectId": "65oh1u"
+    "testFiles": "**/*.feature",
+    "screenshots": {
+        "screenshotOnRunFailure": false
```

Comment:
Why disable this?

Reply:
I think this should be explicitly turned on for CI. During development this won't be needed as we're probably debugging in the browser anyways

Reply:
Why remove the `projectId` for the Cypress Dashboard?

Reply:
It's just at the bottom of the file